torch/csrc/jit/codegen/cuda/kernel_cache.cpp

```diff
@@ -289,40 +290,25 @@ FusionKernelRuntime::FusionKernelRuntime(
     // Take ownership and segment transformed fusion
```

Comment:
Should mostly be code movement and removal of duplicate logic for what used to be the single-kernel path.